pkgs/python-utils/default.nix

```
@@ -32,19 +32,19 @@ let
       ldLibraryPathConvertWrapper = pkgs.writeShellApplication {
         inherit name;
         text = ''
-          if test "''${REPLIT_RTLD_LOADER:-}" = "1" && test "''${REPLIT_NIX_CHANNEL:-}" != "legacy"
+          if test "''${REPLIT_NIX_CHANNEL:-}" = "legacy"
```

Something like this should resolve it, I think:

```suggestion
          if test "''${REPLIT_NIX_CHANNEL:-}" = "legacy" || test "''${REPLIT_NIX_CHANNEL:-}" = "stable-21_11"
```